synapse/handlers/room.py

```diff
@@ -220,6 +222,20 @@ def create_room(self, user_id, room_id, config):
                 "content": {"membership": Membership.INVITE},
             }, ratelimit=False)
 
+        for invite_3pid in invite_3pid_list:
+            id_server = invite_3pid["id_server"]
+            address = invite_3pid["address"]
+            medium = invite_3pid["medium"]
+            yield self.hs.get_handlers().room_member_handler.do_3pid_invite(
+                room_id,
+                user,
+                medium,
+                address,
+                id_server,
+                None,
+                None,
```

Comment:
> Can we use named args here? So that I know what's being set to `None`

Done.

> Will the do_3pid_invite be ratelimited?

It just ends up calling `create_and_send_event` I think, so its probably fiiiiiiiiiine